systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaUserResource.java

```diff
@@ -41,16 +41,15 @@ public static DoneableKafkaUser tlsUser(String clusterName, String name) {
 
     public static DoneableKafkaUser scramShaUser(String clusterName, String name) {
         return user(defaultUser(clusterName, name)
-            .editSpec()
+            .withNewSpec()
                 .withNewKafkaUserScramSha512ClientAuthentication()
                 .endKafkaUserScramSha512ClientAuthentication()
             .endSpec()
             .build());
     }
 
     public static KafkaUserBuilder defaultUser(String clusterName, String name) {
-        KafkaUser kafkaUser = getKafkaUserFromYaml(PATH_TO_KAFKA_USER_CONFIG);
-        return new KafkaUserBuilder(kafkaUser)
+        return new KafkaUserBuilder()
```

Comment:
so you are not using the provided example as starting point anymore but creating a `KafkaUser` from scratch. The above `tlsUser` and `scramShaUser` just add the authentication. How do you deal with tests where authorization is needed now? Do you change the returned user "on fly" to add the authorization part with ACLs?

Reply:
Yes :) We do not have many test cases, where the ACLs are enabled. These test cases will be found in the regression profile )) when we execute them ))